### PR TITLE
Fix results number on standards types page

### PR DESCRIPTION
**Pull request by denl95**
https://trello.com/c/yOsnPVoQ/165-1-t116-fix-search-results-for-type-filtered-standards-1

### CODE REVIEW

**Comment by RevelsGit** [comment]
Wrong results number: https://gyazo.com/0d178473a92d8a6de16369f8be8c255f
The actual results count is 4

File: app/imports/ui/components/standards/standards-list.js [hunk 1]
```diff
@@ -51,8 +51,11 @@ Template.StandardsList.viewmodel({
 
     const standards = Standards.find(standardsQuery).fetch();
```
Comment (RevelsGit) [inline app/imports/ui/components/standards/standards-list.js]: Better to split this into its own function

File: app/imports/ui/components/standards/standards-list.js [hunk 2]
```diff
@@ -76,6 +79,9 @@ Template.StandardsList.viewmodel({
       return typeSections.length > 0;
     });
     
+    const resultsNumber = Standards.find({ typeId: { $in: filteredTypeIds } }).fetch().length;
+    this.searchResultsNumber(resultsNumber);
```
Comment (RevelsGit) [inline app/imports/ui/components/standards/standards-list.js]: It's bad to update reactive values in helpers

Comment (denl95) [inline app/imports/ui/components/standards/standards-list.js]: What can you advice then? Where should I update it?

File: app/imports/ui/components/standards/standards-list.js (hunk 1 above)
Comment (denl95) [inline app/imports/ui/components/standards/standards-list.js]: Can't find that bug like on the screenshot

File: app/imports/ui/components/standards/standards-list.js (hunk 2 above)
Comment (denl95) [inline app/imports/ui/components/standards/standards-list.js]: @RevelsGit 
What can you advice? Where should I update it in this case?

File: app/imports/ui/components/standards/standards-list.js (hunk 1 above)
Comment (denl95) [inline app/imports/ui/components/standards/standards-list.js]: @RevelsGit Can't find this bug. As for me everything is OK

File: app/imports/ui/components/standards/standards-list.js [hunk 3]
```diff
@@ -9,32 +9,24 @@ Template.StandardsList.viewmodel({
...
-        _.each(this.queryStandards(), standard => this.expandCollapsedStandard(standard._id));
-      }
-    },
-    function() {
-      if (!this.searchText() && !!this.selectedStandardId()) {
-        this.expandCollapsedStandard(this.selectedStandardId());
+      if (!!this.searchText() && this.standards().count() > 0) {
+        const ids = this.standards().fetch().map(standard => standard._id);
+
+        this.expandCollapsedStandard(ids, { expandAll: true });
```
Comment (meyginix) [inline app/imports/ui/components/standards/standards-list.js]: This does not work. Function ```expandCollapsedStandard ``` takes only single standard id

Comment (RevelsGit) [inline app/imports/ui/components/standards/standards-list.js]: look at pl1-standards branch. It's all fixed

Comment (meyginix) [inline app/imports/ui/components/standards/standards-list.js]: Expanding works on ```pl1-standards```. So it seems this PR can be closed

**Comment by Praxie** [comment]
It seems to be fixed already

**Pull request closed without merging**